hooks/useControllerPricesData/useControllerPricesData.ts

```diff
@@ -146,20 +146,20 @@ function targets(
 ) {
   const now = Math.floor(Date.now() / 1000);
   const sortedTargets = [...targetUpdates].sort(
-    (a, b) => parseInt(a.timestamp) - parseInt(b.timestamp),
+    (a, b) => a.timestamp - b.timestamp,
```

Comment:
after using new subgraph got a type error here. These are now recognized as numbers and shouldn't be parsed.

Reply:
yup! good catch